php/php.editor/test/unit/data/testfiles/codegen/testTypedPropertiesGetter/testTypedPropertiesGetter.php.testTypedPropertiesGetter_PHP56.codegen

```diff
@@ -0,0 +1,72 @@
+public  function getInstanceArray() {
```

Comment:
We have 2 spaces (`  `) here. This is not nice...

Reply:
Now I see - the formatter is called afterward, right?

Reply:
Yes, they are template text. So actual results (in an editor) should be formatted.